misc/userscripts/qute-pass

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```

Comment:
This should probably be `python3`.

Reply:
Yep, totally forgot not everyone is using a distro with Python 3 as the default...